### ISSUE TITLE
(NEW IDEA) Yet Another ARM request

### ISSUE THREAD

**Issue by Merith-TK** [cdr/code-server#762]
Basically, i think this https://code.headmelted.com/ would be a useful base for code-server and its arm build

**Comment by deansheather** [comment]
I believe the main blocker for ARM builds is getting ARM support into our binary compiler, nbin. There's a PR for this here: https://github.com/cdr/nbin/pull/7.

We build VS Code ourselves when we build code-server, so once we get nbin working with ARM and tested it shouldn't be long before we can get an ARM build out for code-server.

**Comment by Merith-TK** [comment]
Alright,

**Comment by sr229** [comment]
Duplicate of #35, request closure instead.

We won't be planning to use another base for this kind of thing if we already have our own solution.

**Comment by Merith-TK** [comment]
there you go @sr229
